features/measurements/impl/src/test/java/org/opennms/netmgt/measurements/filters/impl/LoadDataHWForecastTest.java

```diff
@@ -0,0 +1,165 @@
+/*******************************************************************************
```

Comment:
copyright headers are not right. check style is complaining about 1st line I guess. Just copy it from other files

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.3.1:check (check-style) on project org.*******.features.measurements.impl: Failed during checkstyle configuration: Exception was thrown while processing /home/circleci/project/features/measurements/impl/src/test/java/org/*******/netmgt/measurements/filters/impl/LoadData.java: IllegalStateException occurred while parsing file /home/circleci/project/features/measurements/impl/src/test/java/org/*******/netmgt/measurements/filters/impl/LoadData.java. 1:0: no viable alternative at input '*': NoViableAltException -> [Help 1]
```

Reply:
Maybe now is fixed. Could you take e look at this pr.
My branch compiles.